packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -102,20 +117,7 @@ const plugin: PrismaVSCodePlugin = {
   activate: async (context) => {
     const isDebugOrTest = isDebugOrTestSession()
 
-    const rootPath = workspace.workspaceFolders?.[0].uri.path
-    if (rootPath) {
-      // This setting defaults to true (see package.json of vscode extension)
-      const isFileWatcherEnabled = workspace.getConfiguration('prisma').get('fileWatcher')
-
-      if (isFileWatcherEnabled) {
-        watcherInstance = startFileWatcher(rootPath)
-        console.debug('File Watcher is enabled and started.')
-      } else {
-        console.debug('File Watcher is disabled.')
-      }
-    } else {
-      console.debug('File Watcher was skipped, rootPath is falsy')
```

Comment:
This special case is not needed any more today?

Reply:
When there's no rootPath, then it falls back to the new and more efficient watcher anyways. So there is always a watching mechanism working.